meteor/server/migration/X_X_X.ts

```diff
@@ -12,5 +15,81 @@ import { CURRENT_SYSTEM_VERSION } from './currentSystemVersion'
  */
 
 export const addSteps = addMigrationSteps(CURRENT_SYSTEM_VERSION, [
-	// Add some migrations here:
+	{
+		id: `convert routesets to ObjectWithOverrides`,
+		canBeRunAutomatically: true,
+		validate: async () => {
+			const studios = await Studios.findFetchAsync({ routeSets: { $exists: true } })
+
+			for (const studio of studios) {
+				//@ts-expect-error routeSets is not typed as ObjectWithOverrides
+				if (studio.routeSets) {
+					return 'routesets must be converted to an ObjectWithOverrides'
+				}
+			}
+
+			return false
+		},
+		migrate: async () => {
+			const studios = await Studios.findFetchAsync({ routeSets: { $exists: true } })
+
+			for (const studio of studios) {
+				//@ts-expect-error routeSets is not typed as ObjectWithOverrides
+				if (!studio.routeSets) continue
```

Comment:
validate is iterating through each studio and making sure they are truthy, so I don't think this could happen.
But I've made the change anyway, to make sure and make it clearer that the code is 'safe'